controlpanel/frontend/jinja2/includes/forms.html

```diff
@@ -56,8 +56,3 @@ <h2 class="heading-medium govuk-fieldset__heading">
     {{ caller() }}
   </fieldset>
 {%- endmacro %}
-
-{% macro delete_button(label, confirm="") -%}
-  <input type="submit" class="button button-warning right js-confirm" value="{{
-  label }}" data-confirm-message="{{ confirm | default("Are you sure?") }}">
-{%- endmacro %}
```

Comment:
I thought this whole file was already deleted - why is this change here?

Reply:
Maybe was deleted recently? It wasn't deleted when I made the change 😆

Reply:
I must have it in a branch somewhere 😕